hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/net/NetUtils.java

```diff
@@ -229,6 +230,23 @@ public static InetSocketAddress createSocketAddr(String target,
     }
     target = target.trim();
     boolean hasScheme = target.contains("://");
+    if (StringUtils.countMatches(target, ":") > 2) {
```

Comment:
NetUtils

Reply:
Thanks for review, fixed as you mentioned